content/operate/rc/changelog/may-2024.md

```diff
@@ -0,0 +1,21 @@
+---
+Title: Redis Cloud changelog (May 2024)
+alwaysopen: false
+categories:
+- docs
+- operate
+- rc
+description: New features, enhancements, and other changes added to Redis Cloud during
+  May 2024.
+highlights: SAML domain verification
+linktitle: May 2024
+weight: 50
+---
+
+## New features
+
+### SAML domain verification
+
+We added domain verification to [SAML single sign-on]({{< relref "/operate/rc/security/access-control/saml-sso" >}}) to enhance security. Before you set up SAML in Redis Cloud, you must first [verify domain ownership]({{< relref "/operate/rc/security/access-control/saml-sso#verify-domain" >}}) for any domains associated with your SAML setup.
```

Comment:
@joniredis What's the guidance for current customers who have already set up SAML? I'd like to add that here.